sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -433,6 +433,61 @@ class JDBCSuite extends QueryTest
     assert(ids(2) === 3)
   }
 
+  test("SPARK-34843: columnPartition should generate a correct stride size") {
```

Comment:
What's a difference between this test and `test("SPARK-34843: columnPartition should realign the first partition for better distribution")`? They have a different test purpose?

Reply:
It was to explicitly test out different parts of the code change. The first test was for proper stride size, the second test was for realignment of the lower bound. However, the second test does implicitly test out both. I've removed the first test since it doesn't add much value and is one more thing to maintain.